lib/ood_core/batch_connect/templates/vnc.rb

```diff
@@ -130,7 +130,8 @@ def run_script
           # successful connections so that the password can be reset
           def after_script
             websockify_cmd = context.fetch(:websockify_cmd, "${WEBSOCKIFY_CMD:-/opt/websockify/run}").to_s
-
+            websockify_hb = context.fetch(:websockify_hb, "${WEBSOCKIFY_HB:-30}").to_s
```

Comment:
I'm wondering if this should be expanded to `websockify_heartbeat_seconds` just to be a little more specific?  The internal variable doesn't matter as much to me, but the user facing configuration I think may need to be a little more explicit?

Reply:
Fine by me. I did consider this, but left it similar to the previous line where command has been abbreviated to 'cmd'

Reply:
I was considering the same lol. `cmd` was before my time, but to an admin configuring it - it may be more recognizable then `hb`.

Reply:
I'm going to go ahead and merge this now and change it later. Thanks again for the contribution!